src/pytest_cov/plugin.py

```diff
@@ -217,6 +217,7 @@ def __init__(self, options, pluginmanager, start=True, no_cov_should_warn=False)
         self._start_path = None
         self._disabled = False
         self.options = options
+        self.wrote_heading = False
```

Comment:
Should probably be private?

```suggestion
        self._wrote_heading = False
```

Reply:
done